test/blackbox/run_blackbox_tests/test_cases_get_node_reachability.c

```diff
@@ -168,11 +181,10 @@ static bool test_get_node_reachability_01(void) {
 	time_t last_unreachable, last_reachable;
 
 	meshlink_set_log_cb(NULL, MESHLINK_DEBUG, log_cb);
-	assert(meshlink_destroy(NUT_CONF));
```

Comment:
I would still destroy `NUT_CONF_01` here, since we don't want any left-over directory from a previous run to interfere with the test.